content/docs/reference/design-docs/train-sim-v3/_index.en.md

```diff
@@ -81,6 +67,62 @@ TrainSim([train simulator])
 SimResults[simulation result curve]
 ```
 
+### Target schedule
+The target schedule is a list of target arrival times at points specified along the path.
+To respect the schedule, the train may have to not use its maximum traction.
+
+### Train state
+The train state is a vector of properties describing the train at a given point in time.
+- position
+- speed
+- position of pantographs 
+- driver reaction times ?
+- battery state ?
+- time elapsed since the last update
+
+### Driving instructions
+
+{% pageinfo color="warning" %}
+This is a copy paste from the driving instructions subpage.
+{% end %}
+
+Driving instructions model what the train has to do, and under what conditions.
+Driving instructions are generated using domain constraints such as:
+
+- unsignaled line speed limits
+- permanent signaled speed limits
+- temporary speed limits
+- dynamic signaling:
+  - block / moving block
+  - dynamically signaled speed restrictions
+- neutral zones
+- stops
```

Comment:
Ref https://github.com/OpenRailAssociation/osrd-website/pull/208 which touches this part as well